kubernetes/linux/Dockerfile.multiarch

```diff
@@ -42,19 +37,98 @@ WORKDIR ${tmpdir}
 RUN chmod 775 $tmpdir/*.sh; sync; $tmpdir/setup.sh ${TARGETARCH}
 
 # Do vulnerability scan in a seperate stage to avoid adding layer

```

Comment:
we should vulnerability scanning in the filesystem which will be part of the final image. so try to do this as the last step

Reply:
The last step is basically taking in all the necessary files. We are not installing anything new. So, in terms of catching vulnerabilities of our packages we will be good. But we might catch false positives here. Let me see if there is a way to remove trivy after the distroless move because we wouldnt want to ship with trivy with the image.